migrations/000005_create_book_properties.up.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
+
+CREATE TYPE BOOK_FORMAT AS ENUM('audiobook', 'e-book', 'paperback', 'hardcover');
+CREATE TYPE BOOK_AVAILABILITY AS ENUM('yes', 'no', 'upcoming');
+
+CREATE TABLE IF NOT EXISTS book_properties(
+    isbn TEXT PRIMARY KEY,
```

Comment:
It should not be the primary key. Text primary keys are extremely inefficient. Make it unique and specify a bigint id instead.